cfme/tests/cloud/test_quota_tagging.py

```diff
@@ -130,14 +130,18 @@ def entities(appliance, request, max_quota_test_instance):
 
 @pytest.fixture(scope='function')
 def set_entity_quota_tag(request, entities, appliance):
-    tag, value = request.param
+    tag_name, value = request.param
     tag = appliance.collections.categories.instantiate(
-        display_name=tag).collections.tags.instantiate(
+        display_name=tag_name).collections.tags.instantiate(
         display_name=value)
     entities.add_tag(tag)
     yield
     # will refresh page as navigation to configuration is blocked if alert are on requests page
     appliance.server.browser.refresh()
+    if entities.appliance.version > '5.11':
```

Comment:
Please add a comment about what is different in 5.11 . Is that a Bug in 5.11 ?

Reply:
You have an appliance object already, no need to reference through entities.